text/0013-abort-pending-updates.md

```diff
@@ -0,0 +1,143 @@
+- Title: Abort pending updates
+- Start Date:
+- Specification PR: https://github.com/meilisearch/specifications/pull/13
+- MeiliSearch Issue: 
+    - https://github.com/meilisearch/MeiliSearch/issues/1104
+- Parent Spec: https://github.com/meilisearch/specifications/pull/12
+
+
+# Abort pending updates
+
+## 1. Feature Description and Interaction
+
+### Summary
+
+Allow to abort messages in the update queue or to empty the entire queue. An aborted update will have the status `aborted`.
+
+Questions not answered:
+- We have an attribute on update info `processedAt`. What becomes this attribute if we add the possibility to abort an update? We already use this term in the case of a failed update.
```

Comment:
I don't see any problem or conflict with this. Aborting an update is a way of processing it, so we could just set the time it was aborted at.

Reply:
The issue exists because we use `processed` as a status to say "it's done". So it could be confused for our users in the context of MeiliSearch since we have chosen this status named `processed`.

Reply:
We will keep both!